src/components/Class/redux/action.js

```diff
@@ -42,4 +42,12 @@ export const actions = {
       data,
     };
   },
+  dropOutClass(data, index) {
+    let item = [...data][index];
+    item.enrolled = false;
```

Comment:
this should not happen here, any mutation, modifying of data should be done in the reducer

Reply:
https://github.com/navgurukul/bhanwari-devi/pull/85/commits/9feab343ff61492a0305dafdf53c22262ccd9e11